arcade/draw_commands.py

```diff
@@ -743,7 +743,7 @@ def draw_rectangle_outline(center_x: float, center_y: float, width: float,
 
 @warning(
     message=\
-        "Drawing with lrtb(left, right, top, bottom) is deprecated use lrbt instead i.e draw_lrbt_rectangle_filled.",
+        "`draw_lrtb_rectangle_filled` is deprecated. Use `draw_lrbt_rectangle_filled` instead.",
```

Comment:
To my understanding, the enclosing backticks are only processed in Sphinx RST, not in ordinary terminal output. Does the warning decorator render them somehow?

Reply:
Doesn't matter if they do or don't do, it still kind of shows that we are mentioning about a specific function.

Reply:
Speaking of sphinx shall I add the warning in the docstring too?

Reply:
Probably not a bad idea

Reply:
Back to the decorator itself. I do agree backticks are not needed in terminal output and it can probably be expressed cleaner like this?
```py
@warning(
    message=((
        "draw_lrtb_rectangle_filled is deprecated. "
        "Use draw_lrbt_rectangle_filled instead."
    )),
    warning_type=NameChangeWarning,
)
```

Reply:
OK! Will do these after lunch.

Reply:
Could also be a better way to design the warning system so you could pass "old_name" and "new_name" somewhere that would generate a generic message from that, but I'm not too familiar with this. It's at least good enough for now and it can easily be tweaked in the future.

Reply:
Will also do that.

Reply:
The decorator also has the function reference, so the name can be obtained from that meaning you only need one argument for the new function that should be used instead?

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1650/commits/d9cdfbe7fe31b201acdedf9f8bdf256abb005bbd